internal/testbed/go.mod

```diff
@@ -5,12 +5,12 @@ go 1.22.6
 require (
 	github.com/Dynatrace/dynatrace-otel-collector/internal/testcommon v0.0.0
 	github.com/google/uuid v1.6.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/testbed v0.108.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/testbed v0.109.1-0.20240917213807-5cb166da005b
```

Comment:
Using the latest commit on main for the testbed for now, as this includes https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/35201 which is needed for the syslog test.